python-backend/tests/mines/resources/test_mine_type_detail_resource.py

```diff
@@ -97,6 +161,31 @@ def test_post_mine_disturbance_duplicate(test_client, auth_headers):
     }
 
 
+def test_post_mine_commodity_duplicate(test_client, auth_headers):
+    test_data = {
+        'mine_type_guid': TEST_MINE_TYPE_GUID,
+        'mine_commodity_code': TEST_MINE_COMMODITY_CODES[1]
+    }
+    post_resp1 = test_client.post('/mines/mine_types/details', data=test_data, headers=auth_headers['full_auth_header'])
+    assert post_resp1.status_code == 200
+
+    post_data1 = json.loads(post_resp1.data.decode())
+    assert post_data1['mine_type_guid'] == test_data['mine_type_guid']
+    assert post_data1['mine_commodity_code'] == test_data['mine_commodity_code']
+    assert post_data1['active_ind'] == True
+
```

Comment:
When database defines the defaults active_ind == true will work as long as this is integration test, which we want to move away from in the future. Consider removing this assert as it is redundant and will fail when we mock the db?

Reply:
The `active_ind = true` is definitely required behaviour upon record creation. I'd feel uncomfortable removing test coverage on this at this point

Reply:
Ok, i'm also unsure if should actually be returning active_ind in our json functions, i started a discussion in slack on that.

Reply:
Saw the slack message. Will remove the `active_ind`. I misunderstood their purpose. I thought they were going to be used for showing historical data in the UI

Reply:
Yea, it's the soft deleted flag. Another item that needs a look when the new db auditing/history is determined.

Reply:
Just a heads-up, it's being returned in a few of the mine documents-related JSON responses as well